### ISSUE TITLE
Specific change relative to the original Aladdin's implementation

### ISSUE THREAD

**Issue by ksmdnl** [johnnylord/trytry-detection#18]
Hello thanks for sharing your work! I could reproduce the performance that you reported using your repository. 
My question though is, which specific changes that you made according to Aladdins implementation https://github.com/aladdinpersson/Machine-Learning-Collection/tree/master/ML/Pytorch/object_detection/YOLOv3 ?

**Comment by johnnylord** [comment]
Aladdins did a great work and I learn a lot from his implementation. I refer his work a lot and just do some miner modification. If you want to know what I done, you can use diff tool by yourself.

**Comment by ksmdnl** [comment]
From what I've seen they are some minor modifications as you said. I'm just interested which changes roughly are, since I could not reproduce the same result using his own repository. Could you perhaps give me a head up just roughly or which part includes the modification?

**Comment by johnnylord** [comment]
I would say that the most tricky part is the loss function. I spent a lot of time to understand it and debug it.

However, I done this work a long date back. I cannot remember all the details.

**Comment by ksmdnl** [comment]
Yeah I was thinking the same thing. In your implementation there's some interesting part in the loss calculation where you multply the target with the iou first before take their bce loss, and the other part where you add apply bce on the box center while mse on their dimension. Could you perhaps briefly explain, what the reason behind this approach was?

**Comment by johnnylord** [comment]
IoU loss is better than bce or mse loss for bounding box. You can refer to related papers for explaination.
